src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -508,9 +508,6 @@ def _get_final_device_map(device_map, pipeline_class, passed_class_obj, init_dic
     # Load each module in the pipeline on a meta device so that we can derive the device map.
     init_empty_modules = {}
     for name, (library_name, class_name) in init_dict.items():
-        if class_name.startswith("Flax"):
-            raise ValueError("Flax pipelines are not supported with `device_map`.")
-
```

Comment:
Why delete it?

Reply:
just so that we can run the fast test with the tiny checkpoints
why wouldn't it work with flax pipelines?

Reply:
accalerate is a Pytorch only library and we rely on accelerate for device map.